### PR TITLE
Add maps to trip show

### PR DESCRIPTION
**Pull request by hoadam**
## Type of change
- [x] New feature
- [x] Bug Fix
- [ ] Test
- [x] Refactor/Style

## Description
Relevant User Story(s):
*Brief summary of the change(s) included in this pull request*
-Debug and resolve merge conflict
-Render maps in trips dashboard

This description closes #

## Check the correct boxes
- [x] This broke nothing
- [ ] This broke some stuff
- [ ] This broke everything

## Testing Changes
- [x] No Tests have been changed
- [ ] Some Tests have been changed
- [ ] All of the Tests have been changed

*Please describe what in the world happened if tests are broken*

## Checklist:
- [ ] My code has no unused/commented out code
- [ ] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code

### CODE REVIEW

File: app/javascript/controllers/accommodations_controller.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  }
+  updateName(name) {
+    document.getElementById("name").value = name
+  }
+  updateLatLon(lat,lng) {
+    document.getElementById("lat").value = lat;
+    document.getElementById("lon").value = lng;
+  }
+  
+}
```
Comment (yainporter) [inline app/javascript/controllers/accommodations_controller.js:44]: Nice Javascript!

File: app/javascript/controllers/activities_controller.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    document.getElementById("address").value = address
+  }
+  updateName(name) {
+    document.getElementById("name").value = name
+  }
+  updateLatLon(lat,lng) {
+    document.getElementById("lat").value = lat;
+    document.getElementById("lon").value = lng;
+  }
+}
```
Comment (yainporter) [inline app/javascript/controllers/activities_controller.js:44]: Great work on these JS controllers

**Review by yainporter — APPROVED** [review]
Wow! Great job figuring out the places Hoa! Thanks for your work!

File: app/javascript/controllers/activities_controller.js (hunk 2 above)
Comment (IgorrMagalhaess) [reply app/javascript/controllers/activities_controller.js:44]: This is awesome!

File: app/controllers/accommodations_controller.rb [hunk 3]
```diff
@@ -16,23 +17,23 @@ def edit
...
 
   def destroy
-    @trip = TripService.trip_details(current_user.id, params[:trip_id])
-    @accommodation = AccommodationService.accommodation_details(current_user.id, params[:trip_id], params[:id])
     AccommodationService.delete_accommodation(current_user.id, params[:trip_id], params[:id])
+    redirect_to trip_path(params[:trip_id])
   rescue => e
     flash[:error] = "Failed to delete accommodation - #{e.message}"
+    redirect_to trip_path(params[:trip_id])
   end
```
Comment (yainporter) [inline app/controllers/accommodations_controller.rb:32]: Sorry these seem to have gotten deleted

**Review by IgorrMagalhaess — APPROVED** [review]
Great job! Thank you Hoa!!

**Pull request merged**